internal/generated/snippets/speech/apiv1/AdaptationClient/create_custom_class.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Code generated by cloud.google.com/go/internal/gapicgen/gensnippets. DO NOT EDIT.
+
+
+package speech_v1
+
+// [START speech_v1_generated_Adaptation_CreateCustomClass_sync]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	speech "cloud.google.com/go/speech/apiv1"
+	"google.golang.org/api/option"
+	speechpb "google.golang.org/genproto/googleapis/cloud/speech/v1"
+)
+
+func createCustomClass(w io.Writer, parent, customClassId string) error {
+	// The custom class parent element
+	// parent = "projects/[PROJECT]/locations/us"
+	// The id for the custom class
+	// customClassId = "passengerships"
+
+	ctx := context.Background()
+	c, err := speech.NewAdaptationClient(ctx,
+		option.WithEndpoint("us-speech.googleapis.com:443"))
```

Comment:
From @amanda-tarafa on `main.go`: This is problematic, as on the config the value was "https://us-speech.googleapis.com/", I know Go won't accept the protocol and requires the port, so you had to do some tweaking to the value in the config to get this to work.
I'll open a discussion internally about this, because apparently different languages accept/require different things.

Reply:
Comment moved to [speech/apiv1/AdaptationClient/create_custom_class.go](https://github.com/googleapis/google-cloud-go/pull/6688/files#diff-ee0271242537214235b0f2e258beb58df03a416814810cd805bb70a8b622438b).